extensions/sns/src/main.rs

```diff
@@ -38,42 +34,67 @@ pub struct SnsOpts {
 /// Subcommands of `dfx sns`
 #[derive(Parser)]
 enum SubCommand {
-    /// Subcommands for working with configuration.
+    /// Deploy an sns directly to a subnet, skipping the sns-wasms canister.
+    /// The SNS canisters remain controlled by the developer after deployment.
+    /// For use in tests only.
     #[command()]
-    Config(SnsConfigOpts),
-    /// Subcommand for creating an SNS.
+    DeployTestflight(DeployTestflightArgs),
+    /// Manage the config file where the initial sns parameters are set.
     #[command()]
-    Deploy(DeployOpts),
+    InitConfigFile(InitConfigFileArgs),
```

Comment:
Might make sense to list this before DeployTestflight, because then, everything is in chronological order.

Reply:
Done!